NetflixClone/Classes/Models/MovieSearchMulti.swift

```diff
@@ -0,0 +1,48 @@
+//
```

Comment:
Let's use `Movie` object to avoid duplicating similar objects.

Reply:
Good idea! Thanks🔥.